tools/system_libs.py

```diff
@@ -982,15 +982,35 @@ class libsanitizer_common_rt_wasm(CompilerRTWasmLibrary, MTLibrary):
   src_glob_exclude = ['sanitizer_common_nolibc.cc']
 
 
-class libubsan_rt_wasm(CompilerRTWasmLibrary, MTLibrary):
-  name = 'libubsan_rt_wasm'
+class SanitizerLibrary(CompilerRTWasmLibrary, MTLibrary):
   depends = ['libsanitizer_common_rt_wasm']
   never_force = True
 
   includes = [['system', 'lib', 'compiler-rt', 'lib']]
-  cflags = ['-std=c++11', '-DUBSAN_CAN_USE_CXXABI']
+  cflags = ['-std=c++11']
+  src_glob = '*.cc'
+
+
+class libubsan_rt_wasm(SanitizerLibrary):
+  name = 'libubsan_rt_wasm'
```

Comment:
Do these libraries need to end in `_wasm`?   I think think that was only need when we have certain parts a library that were wasm only.  In these cases I don't think its needed.

Reply:
In this case, the entire library is wasm only. I think we can wait until we get rid of fastcomp to get rid of the `_wasm` suffix.

Reply:
Also note that this is already in the code. I don't think we should change the name of all sanitizer libraries in this change.

Reply:
OK, lets do that as a followup perhaps.